fairlearn/reductions/moments/bounded_group_loss.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pandas as pd
+import numpy as np
+from .moment import Moment
+from .moment import _REDUCTION_TYPE_LOSS_MINIMIZATION
+
+
+class LossMoment(Moment):
```

Comment:
I have also been meaning to ask.... can we write useful unit tests of these, with carefully chosen X, Y and A?